go-controller/pkg/ovn/policy.go

```diff
@@ -340,28 +337,32 @@ func (oc *Controller) setACLDenyLogging(ns string, nsInfo *namespaceInfo, aclLog
 			// not suppose to happen
 			return fmt.Errorf("failed to find the ACL for pg=%s: %v", portGroupUUID, err)
 		}
-		if _, stderr, err := util.RunOVNNbctl("set", "acl", uuid,
-			fmt.Sprintf("log=%t", aclLogging != ""), fmt.Sprintf("severity=%s", aclLoggingSev)); err != nil {
-			return fmt.Errorf("failed to modify the pg=%s, stderr: %q (%v)", portGroupUUID, stderr, err)
+		if err := updateACLLoggingLevel(uuid, aclLogging); err != nil {
+			return fmt.Errorf("failed to modify the pg=%s, %v", portGroupUUID, err)
 		}
 	}
 
 	return nil
 }
 
-func (oc *Controller) setNamespaceACLLogging(ns string, nsInfo *namespaceInfo, aclLogging string) error {
-	aclLoggingSev := getACLLoggingSeverity(aclLogging)
+func updateACLLoggingLevel(uuid string, aclLogging string) error {
+	_, stderr, err := util.RunOVNNbctl("set", "acl", uuid,
+		fmt.Sprintf("log=%t", aclLogging != ""), fmt.Sprintf("severity=%s", getACLLoggingSeverity(aclLogging)))
+	if err != nil {
+		return fmt.Errorf("stderr: %q, (%v)", stderr, err)
+	}
+	return nil
+}
 
+func (oc *Controller) setNamespaceACLLogging(ns string, nsInfo *namespaceInfo, aclLogging string) error {
```

Comment:
I would squash this commit into the one before where you defined `setNamespaceACLLogging` and make a note in the commit message

Reply:
Done.